doc/plugin_agent_svidstore_aws_secretsmanager.md

```diff
@@ -18,6 +18,21 @@ The format that is used to store in a secret the issued identity is the followin
 }
 ```
 
+### Required AWS IAM permissions
+
+This plugin requires the following IAM permissions in order to function:
+
+```
+secretsmanager:DescribeSecret
+secretsmanager:CreateSecret
+secretsmanager:RestoreSecret
+secretsmanager:PutSecretValue
+secretsmanager:DeleteSecret
+kms:Encrypt
+```
+
+Please note that this plugin does not require permission to read secret payloads stored.
```

Comment:
Just a small suggestion to clarify this wording. The way it was phrased could be misinterpreted to imply that the plugin reads the secrets but doesn't need perms to do so.
```suggestion
Please note that this plugin does not read secrets it has stored and therefore does not require read permissions.
```